HW02_A_Graph-Fails.Rmd

```diff
@@ -55,33 +55,38 @@ Second, I wanted to move the legend on top of the graph since there aren't any p
 ```{r}
 mpg %>% 
 ggplot() + 
-  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 2) + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
   theme(legend.direction = "horizontal") + 
-  theme(legend.position = c(5, 40))
+  theme(legend.position = c(0.65, 0.85)) +
+  theme(legend.title = element_blank())
 
 ```
+### Alpha does not change the size, it changes the opacity (0 is transparent, 1 is opaque). We change 'alpha' to 'size' to set size 
+### Experimented with different coordinates until I got the legend where I wanted it
+### Removing legend title wouldn't let us use "none" so "element_blank" worked instead. To completely hide the legend, I could give it a position not seen in the graph (legend.position = c(10000,10000)) or use "none".

```

Comment:
This is interesting, will there be any differences in running time between the two options?

Reply:
I didn't think of that! Using such large position may cause a longer time running or crash the program. Alternatively, if the program tries to fit everything into one space, perhaps it'll scale the the entire figure really small to accommodate for the huge distance between the figure and legend. Maybe using a different coordinate is not the best option